Week3/More JavaScript.js

```diff
@@ -0,0 +1,180 @@
+
+/**************************** More JavaScript ***************************** */
+
+// 1. Create a function that takes 3 arguments and returns the sum of the these arguments.
+
+function f1(a,b,c) {
+   return a + b + c
+};
+
+console.log(f1(1,2,3)); //  6
+
+// 2. Create a function named colorCar that receives a color, and prints out, 'a red car' for example.
+
+function colorCar(color) {
+    return "a " + color +" car" 
+}
+
+console.log(colorCar('red'));   //  a red car
+
+// 3. Create an object and a function that takes the object as a parameter and prints out all of its properties and values.
+
+const bestDriver = {
+    firstName:"Mika", 
+    lastName:"Hakkinen", 
+    teamName:"McLaren", 
+    age:50};
+    
+function printObj(obj) {
+    for (const key in obj) {
+        console.log(`His ${key} : ${obj[key]}`)    
+    }
+ 
+} 
+
+printObj(bestDriver);   
+
+
+// 4. Create a function named vehicleType that receives a color, and a code, 1 for car, 2 for motorbike. 
+    // And prints 'a blue motorbike' for example when called as vehicleType("blue", 2)
+
+let vehicleCategory = {
+    1 : "car",
+    2 : "motorbike", 
+};
+    function vehicleType(color,category) {
```

Comment:
Ah, clever! You could also use an array, but this is also fine! But what if you do `vehicleType("green", 3)` ? What happens then?

Reply:
I make some changes and added console.log for the undefined category inputs.